code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm

```diff
@@ -107,4 +107,27 @@
 	display_name = "skrell gill cover"
 	path = /obj/item/clothing/mask/breath/skrell
 	whitelisted = list(SPECIES_SKRELL)
-	sort_category = "Xenowear - Skrell"
\ No newline at end of file
+	sort_category = "Xenowear - Skrell"
+
+/datum/gear/accessory/skrell_passport
+	display_name = "jargon federation passport"
+	path = /obj/item/clothing/accessory/badge/passport/jargon
+	sort_category = "Xenowear - Skrell"
+	whitelisted = list(SPECIES_SKRELL, SPECIES_VAURCA_WARRIOR, SPECIES_VAURCA_WORKER, SPECIES_DIONA)
+	cost = 0
+	flags = 0
+
+// the whitelisted list ensures only people with skrell, vaurca, or diona whitelists can reach this check
+/datum/gear/accessory/skrell_passport/check_species_whitelist(mob/living/carbon/human/H)
+	var/static/list/species_list = list(SPECIES_SKRELL, SPECIES_VAURCA_WARRIOR, SPECIES_VAURCA_WORKER, SPECIES_VAURCA_BREEDER, SPECIES_DIONA)
```

Comment:
i don't think this is explicitly needed now that humans have been removed, but i would like to keep it for the breeder + to keep it in mind for future PRs + to possibly add humans in the future if the human lore team wants to begin integrating with skrell